lib/posix-futex/tests/test_posix_futex.c

```diff
@@ -109,7 +109,7 @@ static int futex(uint32_t *uaddr, int futex_op, uint32_t val,
  * Wait for the futex value to be equal to the iteration number and then
  * increment a variable.
  */
```

Comment:
The futex-tests depend on the syscall-shim library. Hence can you please add this dependancy in the Config.uk file.

Reply:
done